build

```diff
@@ -9,7 +9,6 @@ if [ ! -h gopath/src/${REPO_PATH} ]; then
 fi
 
 export GOBIN=${PWD}/bin
-export GOPATH=${PWD}/gopath
```

Comment:
One thing, while this should be fine because we're using the go 1.6 vendoring, this was previously needed to ensure that it wasn't polluting the local development environment.  Defer to @colhom on this.